docs/content/guides/recipes/howto-private-registry/snippets/env-complete.bicep

```diff
@@ -0,0 +1,54 @@
+//SECRETSTORE
+import radius as radius
+
+@description('Required value, refers to the personal access token or password of the git platform')
+@secure()
+param pat string
+
+resource secretStoreGit 'Applications.Core/secretStores@2023-10-01-preview' = {
+  name: 'my-git-secret-store'
+  properties: {
+    type: 'generic'
+    data: {
+      pat: {
+        value: pat 
+      }
+    }
+  }
+}
+//SECRETSTORE
+
+//ENV
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
```

Comment:
Could you make sure this matches the other secret store please? Even though it's not visible its important we're consistent.